js/main.js

```diff
@@ -21,6 +26,20 @@ function Main(){
   this.isStarted = false;
 }
 
+//Function for stopping and starting audio playback -Z
+function MusicSwitch(){
```

Comment:
@zseiman rename this to `musicSwitch`. This is because in most programming languages a capitalized name implies a class (see `Game` and `Main`).